docs/changelog_3_4_0.rst

```diff
@@ -1,5 +1,121 @@
+- Fixed an ungraceful error being raised when running ``[p]traceback`` with closed DMs (:issue:`4329`)
+- Fixed errors that could arise from invalid URLs (:issue:`4437`)
+- Fixed error being raised with ``[p]set nickname`` when no nickname was provided (:issue:`4451`)
+- Added ``[p]set api list`` to list all currently set API services, without tokens (:issue:`4370`)
+- Added ``[p]set api remove`` to remove API services, including tokens (:issue:`4370`)
+
+Audio
+*****
+
+- Removed lavalink logs from being added to backup (:issue:`4453`, :issue:`4452`)
+
+Custom Commands
+***************
+
+- Fixed ungraceful error being thrown on ``[p]cc edit`` (:issue:`4325`)
+
+Dev
+***
+
+- Added ``[p]repl pause`` to pause/resume the REPL session in the current channel (:issue:`4366`)
+
+Economy
+*******
+
+- Added embed option for ``[p]leaderboard`` (:issue:`4184`, :issue:`4104`)
+
+General
+*******
+
+- Fixed issues with text not being properly URL encoded (:issue:`4024`)
+
+Mod
+***
+
+- Fixed nicknames not being properly stored and logged (:issue:`4131`)
+- Added ``[p]modset mentionspam strict`` allowing for duplicated mentions to count to mention spam cap (:issue:`4359`)
+- Fixed plural typos in ``[p]userinfo`` (:issue:`4397`, :issue:`4379`)
+
+Trivia Lists
+************
+
+- Fixed incorrect order of Machamp and Machoke questions (:issue:`4424`)
+- Added new MLB trivia list (:issue:`4455`)
+
+Warnings
+********
+
+- Fixed users being able to warn users above them in hierarchy (:issue:`4100`)
+- Added bool arguments to toggle commands to improve consistency (:issue:`4409`)
+
+Developer changelog
+-------------------
+
+Core
+****
+
+- Added ``bot.remove_shared_api_services`` to remove all keys and tokens associated with an API service (:issue:`4370`)
+- Added option to return all tokens for an API service if ``service_name`` is not specified in ``bot.get_shared_api_tokens`` (:issue:`4370`)
+
+Bank
+****
+
+- Bank API methods now throw TypeError if a non-integer amount is supplied (:issue:`4376`)
+
+Mod
+***
+
+- Deprecated ``redbot.core.utils.mod.is_allowed_by_hierarchy`` (:issue:`4435`)
+
+Modlog
+******
+
+- Added option to accept a ``discord.Object`` in case creation (:issue:`4326`)
+- Added ``last_known_username`` parameter to ``create_case`` function (:issue:`4326`)
+- Fixed error being raised with a deleted channel in ``case.message_content`` (:issue:`4415`)
+
+Utility
+*******
+
+- Added ``redbot.core.utils.get_end_user_statement`` to attempt to fetch a cog's End User Data Statement (:issue:`4404`)
+- Added ``redbot.core.utils.chat_formatting.quote`` to quote text in a message (:issue:`4425`)
+
+Documentation changes
+---------------------
+
+Config
+******
+
+- Added custom group documentation and tutorial (:issue:`4416`, :issue:`2896`)
+
+Modlog
+******
+
+- Clarified that naive ``datetime`` objects will be treated as local times for paramaters ``created_at`` and ``until`` in ``create_case`` (:issue:`4389`)
```

Comment:
This should link to relevant docs:
```suggestion
- Clarified that naive ``datetime`` objects will be treated as local times for parameters ``created_at`` and ``until`` in `modlog.create_case()` (:issue:`4389`)
```
Also, I found a typo so I included a fix for that in the suggestion above.